src/notifications.c

```diff
@@ -369,21 +369,26 @@ void ConfigChangedCallback(RedisModuleCtx *ctx, RedisModuleEvent eid, uint64_t e
   }
 }
 
-void Initialize_KeyspaceNotifications(RedisModuleCtx *ctx) {
-  RedisModule_SubscribeToKeyspaceEvents(ctx,
-    REDISMODULE_NOTIFY_GENERIC | REDISMODULE_NOTIFY_HASH |
-    REDISMODULE_NOTIFY_TRIMMED | REDISMODULE_NOTIFY_STRING |
-    REDISMODULE_NOTIFY_EXPIRED | REDISMODULE_NOTIFY_EVICTED |
-    REDISMODULE_NOTIFY_LOADED | REDISMODULE_NOTIFY_MODULE,
-    HashNotificationCallback);
+void Initialize_KeyspaceNotifications() {
+  if (!RS_KeyspaceEvents_Initialized) {

```

Comment:
```suggestion
  static bool RS_KeyspaceEvents_Initialized = false;
  if (!RS_KeyspaceEvents_Initialized) {

```
You can put `RS_KeyspaceEvents_Initialized` here as a static variable and then you do not need to define it as global.

Reply:
Right 👍